src/uniparc/components/entry/XRefsFacets.tsx

```diff
@@ -1,123 +1,155 @@
 import { useMemo, FC, ReactNode } from 'react';
 import { countBy, uniqBy } from 'lodash-es';
-import { Facets, Facet } from 'franklin-sites';
+import { Facets, Loader } from 'franklin-sites';
 
 import { EntryTypeIcon } from '../../../shared/components/entry/EntryTypeIcon';
 
 import { EntryType } from '../../../uniprotkb/adapters/uniProtkbConverter';
 
 import {
-  UniParcXRef,
   databaseToEntryType,
+  UniParcAPIModel,
+  UniParcXRef,
+  XRefsInternalDatabases,
 } from '../../adapters/uniParcConverter';
-import { FacetValue } from '../../../uniprotkb/types/responseTypes';
+import {
+  FacetObject,
+  FacetValue,
+} from '../../../uniprotkb/types/responseTypes';
 import { OrganismData } from '../../../uniprotkb/adapters/namesAndTaxonomyConverter';
+import { UseDataAPIWithStaleState } from '../../../shared/hooks/useDataApiWithStale';
+
+import '../../../shared/components/results/styles/results-view.scss';
 
-const sortByCount = (a: FacetValue, b: FacetValue) => b.count - a.count;
+const reviewed: XRefsInternalDatabases = 'UniProtKB/Swiss-Prot';
+const unreviewed: XRefsInternalDatabases = 'UniProtKB/TrEMBL';
+const exceptions = new Set<string>([reviewed, unreviewed]);
+
+const sortByCount = (a: FacetValue, b: FacetValue) => {
+  if (exceptions.has(a.value) && exceptions.has(b.value)) {
+    return a.value === reviewed ? -1 : 1;
+  }
+  if (exceptions.has(a.value)) {
+    return -1;
+  }
+  if (exceptions.has(b.value)) {
+    return 1;
+  }
+  return b.count - a.count;
+};
 
 type OrganismTuple = [taxon: OrganismData, count: number];
 
-const XRefsFacets: FC<{ xrefs?: UniParcXRef[] }> = ({ xrefs }) => {
-  const taxonomyFacetValues = useMemo(() => {
-    const organisms = xrefs
-      ?.map((xref) => xref.organism)
-      .filter((organism: OrganismData | undefined): organism is OrganismData =>
-        Boolean(organism)
-      );
-    const taxonObjects = uniqBy<OrganismData>(
-      organisms,
-      (organism) => organism.taxonId
+const xrefsToFacets = (xrefs?: UniParcXRef[]): FacetObject[] => {
+  if (!xrefs?.length) {
+    return [];
+  }
+  const organisms = xrefs
+    .map((xref) => xref.organism)
+    .filter((organism: OrganismData | undefined): organism is OrganismData =>
+      Boolean(organism)
     );
-    const taxonCounts = countBy<OrganismData>(
-      organisms,
-      (organism) => organism.taxonId
-    );
-    return taxonObjects
-      .map((taxon): OrganismTuple => [taxon, taxonCounts[taxon.taxonId]])
-      .sort((tupleA, tupleB) => tupleB[1] - tupleA[1]);
-  }, [xrefs]);
+  const taxonObjects = uniqBy<OrganismData>(
+    organisms,
+    (organism) => organism.taxonId
+  );
+  const taxonCounts = countBy<OrganismData>(
+    organisms,
+    (organism) => organism.taxonId
+  );
+  const taxonomyFacetValues = taxonObjects
+    .map((taxon): OrganismTuple => [taxon, taxonCounts[taxon.taxonId]])
+    .sort((tupleA, tupleB) => tupleB[1] - tupleA[1]);
```

Comment:
Stylistic suggestion:
```suggestion
    .sort(([, countA], [, countB]) => countB - countA);
```